meta/src/main/scala/meta/parser/rise/Expr.scala

```diff
@@ -0,0 +1,30 @@
+package meta.parser.rise
+
+import fastparse.ScalaWhitespace._
+import fastparse._
+import meta.parser.shared.Identifier
+
+object Expr {
+  sealed trait AST
+  object AST {
```

Comment:
These are meant to be extended in the future, but I see your point that these are Declarations, rather than Expressions.
I will rename this into `Decl`.